common/modules/token_send/src/token_send.rs

```diff
@@ -3,49 +3,91 @@
 elrond_wasm::imports!();
 elrond_wasm::derive_imports!();
 
+extern "C" {
+    fn transferESDTNFTExecute(
```

Comment:
why don't you implement the one with managed buffers ? managedMultiTransferESDTNFTExecute for example ?

Reply:
indeed it is much cleaner with managedMultiTransferESDTNFTExecute, thanks for the suggestion